.github/workflows/bump-images.yaml

```diff
@@ -0,0 +1,104 @@
+name: Bump images on tag push
+on:
+  push:
+    tags:
+      - "[0-9]+.[0-9]+.[0-9]+"
+      - "[0-9]+.[0-9]+.[0-9]+-*"
+
+permissions:
+  id-token: write # This is required for requesting the JWT token
+  contents: read # This is required for actions/checkout
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    outputs:
+      tag: ${{ steps.tag.outputs.tag }}
+    steps:
+      - id: tag
+        # if: github.event_name == 'push' && github.ref_type == 'tag'
+        run: echo "tag=${{ github.ref_name }}" >> $GITHUB_OUTPUT
+
+  build-validator:
+    needs: setup
+    uses: kyma-project/test-infra/.github/workflows/image-builder.yml@main
+    with:
+      name: central-application-connectivity-validator
+      dockerfile: Dockerfile
+      context: components/central-application-connectivity-validator
+      tags: |
+        ${{ needs.setup.outputs.tag }}
+
+  build-cra:
+    needs: setup
+    uses: kyma-project/test-infra/.github/workflows/image-builder.yml@main
+    with:
+      name: compass-runtime-agent
+      dockerfile: Dockerfile
+      context: components/compass-runtime-agent
+      tags: |
+        ${{ needs.setup.outputs.tag }}
+
+  build-gateway:
+    needs: setup
+    uses: kyma-project/test-infra/.github/workflows/image-builder.yml@main
+    with:
+      name: central-application-gateway
+      dockerfile: Dockerfile
+      context: components/central-application-gateway
+      tags: |
+        ${{ needs.setup.outputs.tag }}
+
+  bump-images:
+    runs-on: ubuntu-latest
+    needs: setup
+    env:
+      version: ${{ needs.setup.outputs.tag }}
+      images: >-
+        [
+            "europe-docker.pkg.dev/kyma-project/prod/application-connector-manager",
+            "europe-docker.pkg.dev/kyma-project/prod/central-application-connectivity-validator",
+            "europe-docker.pkg.dev/kyma-project/prod/central-application-gateway",
+            "europe-docker.pkg.dev/kyma-project/prod/compass-runtime-agent"
+        ]
+    steps:
+      - uses: actions/checkout@v4
+      - name: Prepare git
+        run: |
+          git config user.name "ghabot"
+          git config user.email "ghabot@example.com"
+
+          git fetch origin
+          git checkout -f autobump/sec-scanners-config
+          git reset --hard origin/main
+      - name: Bump security images
+        run: |
+          # clear BDBA list
+          yq -i '.bdba = []' ./sec-scanners-config.yaml
+
+          # add latest and versioned images to BDBA list
+          yq -P -i \
+              '(env(images) | map(. + ":latest")) as $latest
+              | (env(images) | map(. + ":" + strenv(version))) as $versioned
+              | .bdba |= $latest + $versioned
+              | .bdba |= sort' ./sec-scanners-config.yaml
+
+      - name: Bump application-connector.yaml images
```

Comment:
Just a tiny request - Could you move the `Bump application-connector.yaml` before `- name: Bump security images`? While it won't change the final result, it's IMO a little bit more logical as bumping the `security file` doesn't make sense without bumping `application-connector.yaml`